organizations/data.py

```diff
@@ -392,10 +422,11 @@ def linkage_to_pair(linkage):
 
     # The set of org-course linkages that we must ENSURE ARE ACTIVE
     # is the set of linkages that were REQUESTED
-    # minus the set of linkages that WILL BE CREATED.
+    # minus the set of linkages that WILL BE CREATED,
+    # unless activate==False, in which case we won't ensure anything is active.
     linkage_pairs_to_ensure_active = (
         requested_linkage_pairs - linkage_pairs_to_create
-    )
+    ) if activate else set()
```

Comment:
Nit: Minor style point, but please avoid the `x = {foo} if {bar} else {blah}` pattern unless you can fit it in one line. If I was scanning this code segment, I think there's a good chance I would have missed the back half of it.

Reply:
Good point. I changed both of the inline ifs to be if-else blocks.